### ISSUE TITLE
Training details for EDSR(x1) ?

### ISSUE THREAD

**Issue by lisosia** [XuecaiHu/Meta-SR-Pytorch#15]
Hi,
I could not find detailed setup for training EDSR(x1) in the paper.
Would you mind telling me the setup details?

Unknown lists are below:
* Is train dataset an mixture of the same resolution input-output pair which are degraded by sequential operation of downscaling and upscaling with a factor of 1.1 ~ 4.0 ?
* are options of EDSR model set to default (as defined in options.py) ? 
* epochs and learning rate? 

thank you.

**Comment by XuecaiHu** [comment]
1.  yes. bicubic down and then upscaling to get the training pairs.
2.  no, you should change the settings in option.py by following the EDSR paper
3. same as our Meta-RDN

when I have time, i will add the training and test code soon.
Thank you for your good question.

**Comment by lisosia** [comment]
Thank you very much for your answer!
train/test code would be very helpful 👍

**Comment by matri123** [comment]
Hi, the upscaling process uses the bicubic interpolation or other interpolation methods?

**Comment by lisosia** [comment]
reopened by mistake
